pkg/apis/provisioning/v1alpha5/labels.go

```diff
@@ -34,6 +34,7 @@ var (
 	DoNotEvictPodAnnotationKey        = Group + "/do-not-evict"
 	DoNotConsolidateNodeAnnotationKey = Group + "/do-not-consolidate"
 	EmptinessTimestampAnnotationKey   = Group + "/emptiness-timestamp"
+	DriftLabelKey                     = Group + "/node-drifted"
```

Comment:
Didn't we decide that this should be `karpenter.sh/voluntary-disruption=drifted`

Reply:
Should be an annotation, too, I think.

Reply:
Discussed with Jonathan offline about this. Decided to follow how we're doing other methods of deprovisioning annotation and then changing them all at once when we change to that semantic. WDYT?

Reply:
Discussed offline: reverted to `karpenter.sh/voluntary-disruption=drifted`